include/hdmi_cec_driver.h

```diff
@@ -136,7 +132,8 @@ typedef void (*HdmiCecTxCallback_t)(int handle, void *callbackData, int result);
  * @retval HDMI_CEC_IO_INVALID_ARGUMENT           - Parameter passed to this function is invalid
  * @retval HDMI_CEC_IO_LOGICALADDRESS_UNAVAILABLE - Logical address is not available for source devices. 
  * 
- * @note For sink devices logical address discovery will not happen in HdmiCecOpen()
+ * @note For HDMI source devices only, the logical address discovery also occurs in HdmiCecOpen() and
+ * can be retrieved from HdmiCecGetLogicalAddress().
  * 
  * @warning This API is NOT thread safe.
```

Comment:
add this as a post condition for HdmiCecOpen()
`@post HdmiCecClose() must be called to release resources.`

Reply:
updated